app/views/user_mailer/claim_notification.text.erb

```diff
@@ -1,30 +1,34 @@
 <% content_for :message do %>
-  <%= t("mailer.general.greeting.introductory") %>
+<%= t("mailer.general.greeting.introductory") %>
 
-  You're receiving this e-mail because you had works in a fanworks archive that has been imported by Open Doors (http://opendoors.transformativeworks.org) into the Archive of Our Own (AO3 - <%= root_url %>). Because this e-mail address is connected to one registered on the imported archive, the associated fanworks (listed below) have been automatically added to your AO3 account.
+<%= t(".introduction.text", open_doors_url: "http://opendoors.transformativeworks.org", app_url: root_url) %>
 
-  You can read announcements about recent archive moves at AO3 News (<%= root_url %>admin_posts?tag=18), and find additional information on Open Doors's FAQ page (http://opendoors.transformativeworks.org/faq) or tutorials page (http://opendoors.transformativeworks.org/tutorials). For any questions not answered in the FAQ, tutorials, or this e-mail, please contact AO3 Support at <%= root_url %>support.
+<%= t(".more_info.text", news_url: admin_posts_url(tag: 18), open_doors_faq_url: "http://opendoors.transformativeworks.org/faq", open_doors_tutorial_url: "http://opendoors.transformativeworks.org/tutorials", support_url: new_feedback_report_url) %>
 
-  If this is a mistake and these are not your works, please don't delete them! Please contact Open Doors (https://opendoors.transformativeworks.org/en/contact-open-doors/) and we will sort it out.
+<%= t(".mistake.text", open_doors_url: "https://opendoors.transformativeworks.org/en/contact-open-doors/") %>
 
-  Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please contact Support.
+<%= t(".access.text", support_url: new_feedback_report_url) %>
 
-  These works were written under the e-mail: <%= @external_email %>
-  <% @claimed_works.each do |work| %>
-    - <%= work.title.html_safe %>  <%= work_url(work) %> <%= (work.fandom_string.blank? ? "" : " (#{work.fandom_string})") %>
+<%= t(".works_by", email: @external_email) %>
+<% @claimed_works.each do |work| %>
+  <% if work.fandom_string.present? %>
+    <%= t(".work_info.text.with_fandom", work_title: work.title, work_url: work_url(work), fandom: work.fandom_string.split(",").to_sentence) %>
```

Comment:
This indent and the one in line 17 also need to be removed.

Reply:
Thanks, I hope it's right this time?

Reply:
Yep, thank you!